### PR TITLE
Image upload front end

### PR DESCRIPTION
**Pull request by SabrinaSnider**
Finished implementing blobstore by linking the front-end code with the servlets. Wrote the fetch functions for getting the blobstore upload URL and posting to it when submitting a comment. Changed the submitComment method to account for images.

image

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java [hunk 1]
```diff
@@ -20,19 +32,54 @@
...
 
     newComment.setProperty("email", email);
     newComment.setProperty("timestamp", timestamp);
     newComment.setProperty("message", message);
+    image.ifPresent(url -> newComment.setProperty("image", url));
 
     DatastoreServiceFactory.getDatastoreService().put(newComment);
   }
+
+  /** Returns the URL to the uploaded file or null if the user didn't upload a file. */
```
Comment (ccyang314) [inline portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java:54]: nit: use `@param` and `@return` JavaDoc block tag for commenting the parameter and return type. 

https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-block-tags

File: portfolio/src/main/webapp/scripts/comments.js [hunk 2]
```diff
@@ -33,6 +34,15 @@ function createElementWithParams (tag, {
   return el;
 }
 
+/* */
```
Comment (ccyang314) [inline portfolio/src/main/webapp/scripts/comments.js:37]: nit: add JS function header

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -502,7 +502,7 @@ a:hover {
 }
 
 #comment-form {
-  display: none;
+  /* display: none; */
```
Comment (ccyang314) [inline portfolio/src/main/webapp/style.css:505]: nit: remove commented out code

File: portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java [hunk 4]
```diff
@@ -20,19 +32,54 @@
   /* Adds the request comment to the datastore */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("add comment servelt");
```
Comment (brucezheng) [inline portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java:35]: Delete this?

File: portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java (hunk 1 above)
Comment (brucezheng) [reply portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java:54]: I don't think it's necessary to add the block tags personally. But you should probably update to reflect the Optional return value.

**Pull request merged**